packages/react-components/src/components/VideoEffects/VideoBackgroundEffectsPicker.tsx

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IStyle, Label, mergeStyles, Stack } from '@fluentui/react';
+import React from 'react';
+import { chunk } from '../utils';
+import { _VideoEffectsItem, _VideoEffectsItemProps } from './VideoEffectsItem';
+
+/**
+ * Props for {@link _VideoBackgroundEffectsPicker}
+ * @internal
+ */
+export interface _VideoBackgroundEffectsPickerProps {
+  /**
+   * The options to display in the picker.
+   */
+  options: _VideoBackgroundEffectChoiceOption[];
+
+  /**
+   * The key of the current selected Video Background Effect.
+   * If you provide this, you must maintain selection state by observing onChange events and passing a new value in when changed.
+   */
+  selectedEffectKey?: string;
```

Comment:
Nice controlled component concept such that if this is not defined the component will manage the state but if defined the user must manage the state through the onChange event.